src/WikiService.Data.Provider/WikiService.Data.Provider.csproj

```diff
@@ -6,4 +6,8 @@
 		<ProjectReference Include="..\WikiService.Models.Db\WikiService.Models.Db.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.61" />
```

Comment:
in targets

Reply:
When I use version 1.1.63, I get error in stranger code (in EditArticleCommand).